package.json

```diff
@@ -45,6 +45,6 @@
     "node": ">= 4.0.0"
   },
   "scripts": {
-    "test": "./node_modules/vows/bin/vows test/*.js --spec"
+    "test": "vows test/*.js --spec"
```

Comment:
Why change this? Before, it was self-contained.   Doesn't this change create a new dependency on a global installation of "vows"?

Reply:
There is no need to install vows globaly. When run "npm run test" node will search "vows" comand in <project>/node_modules/.bin/vows.sh (or vows.cmd for Windows) first.